src/main.ts

```diff
@@ -515,7 +515,6 @@ export class Purchases {
 
     const certainHTMLTarget = resolvedHTMLTarget as unknown as HTMLElement;
 
-    const asModal = !htmlTarget;
```

Comment:
@nicfix removing asModal here as I have removed backdrop etc (since it's always a 100vwx100vh page)

Reply:
@alfondotnet this was needed to distinguish when we are taking the entire screen from when we are showing it in an element passed by the developer.

We should not lose the functionality.
Using 100vw x 100vh would break whoever is using that mode.

Reply:
as discussed IRL, will not tackle this for this PR as the mode is currently broken. have a WIP fixing this holistically and adding tests